dreammall-earth/dreammall.earth
# feat(backend): error reporting with sentry

Motivation
----------
This will implement 1/2 of #2504: The backend part.

I've chosen Sentry as already communicated. Right now, I'm using their free plan with only 1 member per team. Account credentials are checked into our secrets repository.

There are 2 options if we want more team members:
1. We pay their business plan
2. We self-host: https://github.com/getsentry/self-hosted

How to test
-----------
1. Log in to our Sentry account (you need access to our IT4C secrets repository)
2. Run the following:
```sh
 curl --request POST \
  --header 'content-type: application/json' \
  --url 'https://sentry.git.dreammall.earth/api/' \
  --data '{"query":"query { getTableName(tableId: 1) }"}'
```
3. You will see an error in the dashboard under project `dreammall-backend`
